pkgs/development/python-modules/importlib-resources/2.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, toml
+, importlib-metadata
+, typing ? null
```

Comment:
? null needs to be removed, unless the  nulls are taken care of. In this case, those packages would only be added if Python < version. Since this expression is for Python 2 only, the ? can simply be removed.